keyboards/bastardkb/charybdis/3x5/rules.mk

```diff
@@ -0,0 +1,11 @@
+# Build Options
+#   change yes to no to disable
+#
+RGB_MATRIX_SUPPORTED = yes  # RGB matrix is supported and enabled by default
+RGBLIGHT_SUPPORTED = yes    # RGB underglow is supported, but not enabled by default
```

Comment:
These have been previously removed from the repo, and should not be re-introduced.
```suggestion
```